src/pudl/io_managers.py

```diff
@@ -568,12 +568,17 @@ def __init__(self, base_path: UPath, schema: pa.Schema) -> None:
         super().__init__(base_path=base_path)
         self.schema = schema
 
-    def dump_to_path(self, context: OutputContext, obj: pd.DataFrame, path: UPath):
+    def dump_to_path(self, context: OutputContext, obj: dd.DataFrame, path: UPath):
         """Write dataframe to parquet file."""
-        raise NotImplementedError("This IO Manager doesn't support writing data.")
+        logger.info(f"Write df to parquet at {path}")
+        schema = Resource.from_id("hourly_emissions_epacems").to_pyarrow()
+        obj.repartition(npartitions=1).to_parquet(
+            path, engine="pyarrow", schema=schema, compression="snappy"
+        )
 
-    def load_from_path(self, context: InputContext, path: UPath) -> pd.DataFrame:
+    def load_from_path(self, context: InputContext, path: UPath) -> dd.DataFrame:
```

Comment:
Just flagging that we might want to change the behavior of load_from_path if we define `hourly_emissions_epacems` with partitions. But... maybe it will Just Work, since the `UPathIOManager` has some logic to call `load_from_path` correctly based on the asset partitioning.

Reply:
Ideally we'll use dagster partitions in the future for CEMS which will likely change our IO logic.